### ISSUE TITLE
add Support tab

### ISSUE THREAD

**Issue by certik** [sympy/sympy.github.com#5]
> > Maybe add a "Support" (or some other name) tab that links to the mailing list, IRC channel, and issues tracker.
> 
> I thought it would be on the development tab. So maybe there should be
> a development + support tabs?

I think they should be separate.  If someone has a question or has found a bug in SymPy, they don't necessarily want to join development, especially for new users.  The reverse is not necessarily true either—someone who wants to join development doesn't necessarily want to do so just because they have found a bug that they want to fix.

Of course, some of the information could be on both (like the mailing list, etc.), but I think there should be a support tab that doesn't scare people off with development stuff.  The support tab would also like to the docs, and maybe some more specific stuff like the tutorial, FAQ, and/or the gotchas.

Basically, you will have redundancy, but it is a good thing, because you want to put stuff under headings that they are thinking of when they are looking for something.  So if someone has a problem with SymPy and they go to the website, they are thinking, "I need to find the Support tab," because that is what it is called in most websites. Or if someone thinks, "I want to download the latest development source," about half will try the Downloads tab first and the other half will try the Development tab first, so it is definitely good that we have the git links very clearly there on both.
